setup.py

```diff
@@ -43,6 +43,7 @@ def read_file(filename, encoding='utf8'):
     'pytz',
     'requests',
     'setuptools',  # pkg_resources
+    'six',
```

Comment:
That's a good point about `six` already being installed as a dependency for `acme`. With that in mind, I think it's okay to include it here as well.